src/screens/UserManagement/UserRevamp/UserManagementHelper.res

```diff
@@ -112,10 +113,31 @@ module ProfileSelection = {
     let internalSwitch = OMPSwitchHooks.useInternalSwitch()
     let profileList = Recoil.useRecoilValueFromAtom(HyperswitchAtom.profileListAtom)
     let {userInfo: {userEntity}} = React.useContext(UserInfoProvider.defaultContext)
+    let form = ReactFinalForm.useForm()
+    let formState: ReactFinalForm.formState = ReactFinalForm.useFormState(
+      ReactFinalForm.useFormSubscription(["values"])->Nullable.make,
+    )
 
     let disableSelect = switch userEntity {
     | #Profile => true
-    | #Organization
+    | #Organization => {
+        let selected_merchant =
+          formState.values
+          ->LogicUtils.getDictFromJsonObject
+          ->LogicUtils.getString("merchant_value", "")
```

Comment:
open LogicUtils once and use it

Reply:
it was only two places in the file, so i did not open it on top